tox.ini

```diff
@@ -11,18 +11,15 @@ labels =
 
 [gh-actions]
 python =
-    3.8: py38, black, flake8, mypy, yamllint, bandit, safety, docs, unit_tests
-    3.9: py39, black, flake8, mypy, yamllint, bandit, safety, docs, unit_tests
+    3.12: py312, black, flake8, mypy, yamllint, bandit, safety, docs, unit_tests
 
 [testenv]
 usedevelop = true
-# 3.8 is the current suppported version
-# 3.9 is the latest version available on RHEL8
+# 3.12 is the current supported version on RHEL 8.10
 basepython =
-    py38: python3.8
-    py39: python3.9
-    migrate-db: python3.8
-    pip-compile: python3.8
+    py312: python3.12
```

Comment:
should this also have 3.13?

Reply:
I've added tests for 3.13, please check.